### PR TITLE
Rel Notes doc text for 4.8.9 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
For release 2021.08.31.

2021.08.30: version bumped to 4.8.9

Preview: https://deploy-preview-35867--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes#ocp-4-8-9

### CODE REVIEW

**Review by EricPonvelle — CHANGES_REQUESTED** [review]
Just a small nitpick over a comma. Looks good otherwise!

**Review by EricPonvelle — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2527,14 +2534,32 @@ link:https://access.redhat.com/solutions/6261051[{product-title} 4.8.5 container
...
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6287821[{product-title} 4.8.8 container image list]
+
+[id="ocp-4-8-8-bug-fixes"]
+==== Bug fixes
+
+* This release adds additional localized content in Chinese, Japanese, and Korean. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972987[*BZ#1972987*])
+
+* The address set naming convention used in OVN-Kubernetes for {product-title} 4.5 was changed in {product-title} 4.6, but the migration of existing address sets to the new naming convention was not handled as part of the upgrade. Network policies that were created in version 4.5 with namespace selector criteria for their ingress or egress sections rely on matching old address sets that were not kept up-to-date with the pod IP addresses within such namespaces. These policies might not work correctly in 4.6 or later releases, and might allow or drop unexpected traffic.
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-8-release-notes.adoc]: @jboxman thanks for taking a look! That makes me wonder if it's TMI, but it doesn seem to have some things a user might need to know. I will check in with #forum-sdn

[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-8-release-notes.adoc]: SME acked :+1:

**Comment by sagidlow** [comment]
LGTM.

**Pull request merged**